scrapy/utils/reqser.py

```diff
@@ -1,95 +1,18 @@
-"""
-Helper functions for serializing (and deserializing) requests.
-"""
-import inspect
+import warnings
+from typing import Optional
 
-from scrapy.http import Request
-from scrapy.utils.python import to_unicode
-from scrapy.utils.misc import load_object
+import scrapy
+from scrapy.exceptions import ScrapyDeprecationWarning
+from scrapy.utils.request import request_from_dict  # noqa: F401
 
 
-def request_to_dict(request, spider=None):
-    """Convert Request object to a dict.
+warnings.warn(
+    ("Module `scrapy.utils.reqser` is deprecated, please use `scrapy.Request.to_dict` "
+     " and/or`scrapy.utils.request.request_from_dict` instead"),
+    category=ScrapyDeprecationWarning,
+    stacklevel=2,
+)
```

Comment:
There’s a missing space, and I would use nothing, single quotes or double quotes instead of a [grave accent](https://en.wikipedia.org/wiki/Grave_accent).

```suggestion
warnings.warn(
    ("Module scrapy.utils.reqser is deprecated, please use scrapy.Request.to_dict "
     " and/or scrapy.utils.request.request_from_dict instead"),
    category=ScrapyDeprecationWarning,
    stacklevel=2,
)
```

Reply:
No objections here, thanks. Just as a note, this is done in the [`scrapy.utils.py36`](https://github.com/scrapy/scrapy/blob/2.5.0/scrapy/utils/py36.py#L8) module as well.